backend/lib/endpoints/schema/notifypreference.js

```diff
@@ -0,0 +1,45 @@
+const S = require("fluent-schema");
+const { strictSchema } = require("./utils");
+
+const notifyPreferenceSchema = strictSchema()
+  .prop(
+    "message",
+    S.object()
+      .prop("instant", S.boolean().default(true))
+      .prop("daily", S.boolean().default(false))
+      .prop("weekly", S.boolean().default(false))
+      .prop("biweekly", S.boolean().default(false)),
+  )
+  .prop(
+    "like",
+    S.object()
+      .prop("instant", S.boolean().default(false))
+      .prop("daily", S.boolean().default(false))
+      .prop("weekly", S.boolean().default(true))
+      .prop("biweekly", S.boolean().default(false)),
+  )
+  .prop(
+    "comment",
+    S.object()
+      .prop("instant", S.boolean().default(false))
+      .prop("daily", S.boolean().default(true))
+      .prop("weekly", S.boolean().default(false))
+      .prop("biweekly", S.boolean().default(false)),
+  )
+  .prop(
+    "post",
+    S.object()
+      .prop("instant", S.boolean().default(false))
+      .prop("daily", S.boolean().default(true))
+      .prop("weekly", S.boolean().default(false))
+      .prop("biweekly", S.boolean().default(false)),
+  );
+
+const updateNotifyPrefsSchema = {
+  body: strictSchema()
+    .prop("notifyPrefs", notifyPreferenceSchema),
+};
+
+module.exports = {
+  updateNotifyPrefsSchema,
+};
```

Comment:
Maybe this could be added to `/backend/lib/endpoints/schema/users.js` as part of `updateUserSchema` if you are going to use the same `PATCH /api/users/current` endpoint

Reply:
Right @Zaydme , I was thinking to use this for the `users/unsubscribe` endpoint, in which we don't have full User's data in the request, the user is not authenticated at F/E at that time, and we use a separate token to validate it's the correct user

Reply:
`notifyPreferenceSchema` could be exported and re-used in `updateUserSchema` but also keeping everything in `/backend/lib/endpoints/schema/users.js` would be better

Reply:
that makes sense, I will update it now 👍